apps/opentelemetry/src/otel_configuration.erl

```diff
@@ -100,7 +103,9 @@ new() ->
                    event_count_limit => 128,
                    link_count_limit => 128,
                    attribute_per_event_limit => 128,
-                   attribute_per_link_limit => 128}, t()).
+                   attribute_per_link_limit => 128,
+                   start_default_tracer => true,
```

Comment:
Setting them to `true` by default in order to be compatible with the previous behavior. 
Going to set these both params to `false` in emqx (probably in `emqx_machine` module).